src/univers/versions.py

```diff
@@ -343,3 +344,86 @@ def __gt__(self, other):
         if not isinstance(other, self.__class__):
             return NotImplemented
         return gentoo.vercmp(self.value, other.value) > 0
+
+
+@attr.s(frozen=True, order=False, eq=False, hash=True)
+class LegacyOpensslVersion(Version):
```

Comment:
```suggestion
class OpensslLegacyVersion(Version):
```

Reply:
So in the case of naming, I borrowed it from Philips's suggestion.
https://github.com/nexB/univers/issues/36#issuecomment-1034629474